configure.ac

```diff
@@ -106,6 +106,27 @@ AC_MSG_CHECKING(
     [for vma extra api])
 AC_MSG_RESULT([${have_vma_api}])
 
+##########################################################################
+# check DOCA communication channel API
+#
+AC_ARG_ENABLE(
```

Comment:
it's better to use *AC_ARG_WITH* here to allow specifying alternative library installation path.

Reply:
Pay attention that DOCA should not be configured at the same time with `--with-tls`. See https://github.com/Mellanox/sockperf/blob/sockperf_v2/config/m4/tls.m4#L10
Look at potential issue in this case: https://github.com/Mellanox/sockperf/pull/190/files#diff-649a0b286b520f22da501867c812f1acbc760fe028ea70edb14b60672ba7c716R116